src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -110,6 +110,7 @@
      *
      * Generally this is associated with negative density fluids.
      */
+    @Deprecated
```

Comment:
Leave a javadoc showing how to migrate away from the deprecated thing, something like
`@deprecated This flag is going to be removed, check if the fluid density is negative instead.`